### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by kmilo9999**
solves #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,29 +34,36 @@ WebVR Volume Rendering.
...
 <a name="Demo"/>
 
 #### Live Demo
 
-[Click here for live demo](https://datasci.brown.edu/volume-vr/)
+[Click here for live demo](https://bay-viewer.riddc.brown.edu/)
+
+<a name="UIControls"/>
+
+#### UI Controls
```
Comment (mcmcgrath13) [inline README.md:50]: delete this section? it's already in the app/demo

File: README.md [hunk 2]
```diff
@@ -33,29 +34,36 @@ WebVR Volume Rendering.
...
 ![Application_Arch](./imgs/archit.png)
 
 <a name="Demo"/>
 
 #### Live Demo
 
-[Click here for live demo](https://datasci.brown.edu/volume-vr/)
+[Click here for live demo](https://bay-viewer.riddc.brown.edu/)
+
+<a name="UIControls"/>
```
Comment (mcmcgrath13) [inline README.md:48]: what does this do? is this for the table of contents to target?

**Review by mirestrepo — COMMENTED** [review]
There seems to also be a whole section at the bottom of the readme on `How to access it for the server` that can be deleted, no?

File: README.md [hunk 3]
```diff
@@ -100,51 +90,6 @@ Every component on A-Frame has a series of methods to be extended to determine i
...
-
-![file_system1](./imgs/filesystem3.png)
-
-#### How to access it from the ccv server
-
-Open your web browser and go to https://datasci.brown.edu/spheroid/ or https://datasci.brown.edu/simulation/.
-
-The VR mode is only supported by Mozilla FireFox. Please install this application if you want to use HMD device to visualize the data.
-
 #### How the application works
```
Comment (RobertGemmaJr) [inline README.md:94]: This can be deleted too since the section is gone

```suggestion
```

**Review by RobertGemmaJr — CHANGES_REQUESTED** [review]
The markdown doesn't seem to be working in a few areas. Pretty sure it happens when there's not an empty line above a header

image

image

File: README.md (hunk 1 above)
Comment (kmilo9999) [reply README.md]: I deleted this section

File: README.md (hunk 3 above)
Comment (kmilo9999) [reply README.md]: I added a link to the instructions section in the web page.

**Review by mcmcgrath13 — COMMENTED** [review]
I don't see the images added in this PR being used anymore - can they be deleted?

**Comment by kmilo9999** [comment]
@mcmcgrath13  I deleted the unused images

**Pull request merged**